### ISSUE TITLE
NoMethodError: private method `binary?'

### ISSUE THREAD

**Issue by erickreutz** [rgeo/activerecord-postgis-adapter#164]
Running into this error when trying to create a record with an `st_point` column. The record is still created and returned but the error still happens.

```
Could not log "sql.active_record" event. NoMethodError: private method `binary?' called for #<ActiveRecord::ConnectionAdapters::PostGISAdapter::OID::Spatial:0x007fa5cf54a4f8>
```

**Comment by mrnugget** [comment]
I ran into the same warning and dug into the source. From what I gathered:

1.) ActiveRecord's `Type::Value` defines `binary?` as a public method
2.) PostgreSQL-Adapter `OID`s inherit from `Type::Value` and thus expose the `binary?` method too.
3.) The ActiveRecord LogSubscriber checks `binary?` on each column for logging purposes.
4.) The `OID::Spatial` class in `activerecord-postgis-adapter` makes `binary?` private.
5.) This results in a warning

I'd say the solution is to just make the `binary?`-Method on `OID::Spatial` not private.
